### ISSUE TITLE
Whether to support streaming decode?

### ISSUE THREAD

**Issue by shanhaidexiamo** [charactr-platform/vocos#26]
Hi , thanks for your work. I would like to ask if Vocos can decode with streaming when reconstructing audio from EnCodec tokens？

**Comment by Jackiexiao** [comment]
yes, I try kernel_convnx 3 (instead of 7) which reduces reception field, and with overlap, you can decode with streaming

**Comment by shanhaidexiamo** [comment]
Thanks! So I need to train a new model with kernel_convnx 3.

**Comment by VJJJJJJ1** [comment]
> yes, I try kernel_convnx 3 (instead of 7) which reduces reception field, and with overlap, you can decode with streaming
> 
> kernel_convnx means: depthwise conv's kernel_size`self.dwconv = nn.Conv1d(dim, dim, kernel_size=7, padding=3, groups=dim) `

Do you still set padding=3 in kernel_convnx 3 for streaming decode? 
`self.dwconv = nn.Conv1d(dim, dim, kernel_size=3, padding=3, groups=dim) `
 BTW, how do you modify attnBlock to causal attnBlock?

**Comment by JohnHerry** [comment]
what is the progress? is there any streamable  vocos training config and pipeline?
